<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIBitmapConversion</class>
 <widget class="QWidget" name="UIBitmapConversion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Bitmap file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leInputFilename">
       <property name="text">
        <string>C:/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFilePicker">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/Icons/resources/document-open.png</normaloff>:/Icons/resources/document-open.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item row="0" column="0" rowspan="3">
      <widget class="QLabel" name="imagePreview">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>64</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnPreview">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" rowspan="2" colspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>147</width>
         <height>68</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnGenerateCode">
       <property name="text">
        <string>Generate Code</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="cbColors">
       <item>
        <property name="text">
         <string>Rot</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blau</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Weiss</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gr√ºn</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>378</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="GLCDPreview" name="glcdPreview" native="true">
     <property name="minimumSize">
      <size>
       <width>384</width>
       <height>192</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLCDPreview</class>
   <extends>QWidget</extends>
   <header>glcdpreview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnFilePicker</sender>
   <signal>clicked()</signal>
   <receiver>UIBitmapConversion</receiver>
   <slot>pickInputFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPreview</sender>
   <signal>clicked()</signal>
   <receiver>UIBitmapConversion</receiver>
   <slot>convert2GLCD()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnGenerateCode</sender>
   <signal>clicked()</signal>
   <receiver>UIBitmapConversion</receiver>
   <slot>showCCode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbColors</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>UIBitmapConversion</receiver>
   <slot>setGLCDColor(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pickInputFile()</slot>
  <slot>convert2GLCD()</slot>
  <slot>showCCode()</slot>
  <slot>setGLCDColor(QString)</slot>
 </slots>
</ui>
